version/version.go

```diff
@@ -4,4 +4,4 @@
 
 package version
 
-const defaultBeatVersion = "8.9.0"
+const defaultBeatVersion = "8.8.0"
```

Comment:
[Blocker]
ooops, I think you need to rebase your branch

Reply:
Accidental commit.  Thanks for catching!